docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -1182,17 +1182,19 @@ conditions that vary from the environments individually tested in the various
 components. For example, Accumulo's use of HDFS includes many short block
 reads, which differs from the more common full file read used in most
 map/reduce applications. We have found that certain versions of Accumulo and
-Hadoop will include stability bugs that greatly affect overall stability. In
-our testing, Accumulo 1.6.2, Hadoop 2.6.0, and Zookeeper 3.4.6 resulted in a
-stable VM clusters that did not fail a month of testing, while Accumulo 1.6.1,
-Hadoop 2.5.1, and Zookeeper 3.4.5 had a mean time between failure of less than
-a week under heavy ingest and query load. We expect that results will vary with
-other configurations, and you should choose your software versions with that in
-mind.
-
-
-
-
-
-
-
+Hadoop will include stability bugs that greatly affect overall stability.
+Release notes typically contain information about versions used in
+release testing.
+
+The following table shows the Hadoop, Zookeeper and
+Thrift versions defined in the dependency section of the POM for building the
+artifacts.  
+
+|================================================
+|Accumulo |Hadoop |Zookeeper |Thrift
+|1.7            |2.2.0     |3.4.6          |0.9.1
+|1.8            |2.6.4     |3.4.6          |0.9.3
+|================================================
+
```

Comment:
This simplified table example highlights what I mean about getting stale, with its use of ".x" as a wildcard version. Our dependencies do not follow semver, and we have no idea if the pattern will hold.

> A link to release notes from the user manual is also fine, but that inherently breaks the standalone pdf docs (are we still doing that?).

We publish a standalone user manual, but it's not a PDF. It's a self-contained HTML file. Not sure I understand what you mean by "breaks". A written "redirect" to the website for the most up-to-date information won't break anything.

> I disagree. Most people check the documentation first when they're looking for docs ;). Most people I've met and worked with would not think to look at release notes.

Release notes are docs, too. In any case, we can try to figure out where users are facing, and place stuff into their line of sight... or we can put things where they make the most sense, and tell the users to turn their head to face where we've placed it. I prefer the latter.